modules/core/ante/ante.go

```diff
@@ -8,20 +8,20 @@ import (
 	"github.com/cosmos/ibc-go/v5/modules/core/keeper"
 )
 
-type RedundancyDecorator struct {
+type RedundancyRelayDecorator struct {
```

Comment:
This is maybe a mega nit in wording, but were we going to go with `RedundantRelayDecorator`? 
This is fine by me also, but if using "Redundancy" it might read better as `RelayRedundancyDecorator`.

What do you think? 

Also cc. @colin-axner

Reply:
I think maybe `RelayRedundancyDecorator` would be my preference, but I don't feel strongly about it either way :)

Reply:
I don't have a strong preference either. Happy to let @crodriguezvega decide

Reply:
Sorry, my bad then that I got the name wrong initially. I will rename to `RedundantRelayDecorator`.